plugins/examples/custom-processor/src/main/java/org/elasticsearch/example/customprocessor/ExampleRepeatProcessor.java

```diff
@@ -0,0 +1,48 @@
+package org.elasticsearch.example.customprocessor;
+
+import org.elasticsearch.ingest.AbstractProcessor;
+import org.elasticsearch.ingest.IngestDocument;
+import org.elasticsearch.ingest.Processor;
+
+import java.util.Map;
+
+/**
+ * Example of adding an ingest processor with a plugin.
+ */
+public class ExampleRepeatProcessor extends AbstractProcessor {
+    public static final String TYPE = "repeat";
+    public static final String FILED_TO_REPEAT = "to_repeat";
```

Comment:
I think `FILED_TO_REPEAT` still needs changed to `FIELD_TO_REPEAT`

Reply:
Oh I missed this